yarn-project/circuits.js/src/utils/call_wasm.ts

```diff
@@ -0,0 +1,29 @@
+import { AsyncWasmWrapper } from '@aztec/foundation/wasm';
+import { uint8ArrayToNum } from './serialize.js';
+
+export async function callAsyncWasm<T>(
```

Comment:
I feel like we have loads of these functions around the codebase now.

Reply:
Agree, but hopefully @ludamad's work will standardize all these.